tox.ini

```diff
@@ -1,43 +1,81 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
+minversion = 3.18
 envlist =
-    py{27,py,35,36,37,38}{,-subunit},coverage,lint,docs
+    lint
+    py27
+    py35
+    py36
+    py37
+    py38
+    py39
+    pypy
+    pypy3
+    docs
+    coverage
+    py{27,35,36,37,38,39,py,py3}-subunit
 
 [testenv]
-extras =
-    test
-    subunit: subunit
-commands =
-    python setup.py -q test -q
-
-[testenv:coverage]
 usedevelop = true
-basepython =
-    python3
+deps =
+    zope.testrunner
```

Comment:
I wonder if we should drop this?  Kind of pointless to install zope.testrunner from pypi before reinstalling it from the local source tree.

(This would need another template tweak.)

Reply:
I think it would be worth the additional template tweak.

Reply:
https://github.com/zopefoundation/meta/pull/85